### PR TITLE
Bootstrap 5 changes

### PR DESCRIPTION
**Pull request by joeparsons**
This should be ready for merging into the main (5.x) branch now.

https://getbootstrap.com/docs/5.2/migration/

### CODE REVIEW

**Comment by mmunro-ltrr** [comment]
The whole migration from Bootstrap 4 to 5 might need a spike or epic approach: I'll make an issue to reference that.

**Comment by joeparsons** [comment]
> The whole migration from Bootstrap 4 to 5 might need a spike or epic approach: I'll make an issue to reference that.

Thanks @mmunro-ltrr!

**Comment by mmunro-ltrr** [comment]
That last failure might be a consequence of the file ownership changes the GitHub Actions now do on the checked-out copy of the repository. A quick & dirty fix might be throwing around a `sudo` or two. We've not exercised all the things that could be affected by the ownership changes.

File: package.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
   "name": "arizona-bootstrap",
   "version": "2.0.24",
```
Comment (joeparsons) [inline package.json:3]:
```suggestion
  "version": "5.0.0-alpha1",
```

File: package.json [hunk 2]
```diff
@@ -70,7 +72,7 @@
     "@rollup/plugin-node-resolve": "^15.1.0",
     "autoprefixer": "^10.4.14",
     "babel-plugin-istanbul": "^6.1.1",
-    "bootstrap": "^4.6.2",
+    "bootstrap": "^5.3.2",
```
Comment (joeparsons) [inline package.json:75]:
```suggestion
    "bootstrap": "^5.3.1",
```

File: package.json [hunk 3]
```diff
@@ -70,7 +72,7 @@
     "@rollup/plugin-node-resolve": "^15.1.0",
     "autoprefixer": "^10.4.14",
```
Comment (joeparsons) [inline package.json:73]:
```suggestion
    "autoprefixer": "^10.4.14",
    "@rollup/plugin-replace": "^5.0.2",
```

Comment (joeparsons) [inline package.json:73]:
```suggestion
    "@rollup/plugin-replace": "^5.0.2", 
    "autoprefixer": "^10.4.14",
```

File: scss/arizona-bootstrap.scss [hunk 4]
```diff
@@ -3,46 +3,47 @@
...
- * Copyright 2011-2019 Twitter, Inc.
+ * Bootstrap v5.2.0 (https://getbootstrap.com/)
+ * Copyright 2011-2022 The Bootstrap Authors
+ * Copyright 2011-2022 Twitter, Inc.
  * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
  */
+// TWBS Configuration
 @import "../node_modules/bootstrap/scss/functions";
 @import "../node_modules/bootstrap/scss/variables";
+@import "../node_modules/bootstrap/scss/maps";
```
Comment (joeparsons) [inline scss/arizona-bootstrap.scss:14]:
```suggestion
@import "../node_modules/bootstrap/scss/variables-dark";
@import "../node_modules/bootstrap/scss/maps";
```

**Comment by joeparsons** [comment]
This should be ready for merging into the main (5.x) branch now.

**Review by mmunro-ltrr — APPROVED** [review]
I would be inclined to merge this with the main branch asap: it won't be anything like its final state, but a great way to work on getting there.

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**